packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -12,11 +12,16 @@ flutter:
         pluginClass: PathProviderPlugin
       ios:
         pluginClass: FLTPathProviderPlugin
+      macos:
+        pluginClass: FLTPathProviderPlugin
```

Comment:
Why do we need the plugin class here?

Reply:
we don't https://github.com/flutter/plugins/pull/2569